makefile

```diff
@@ -0,0 +1,14 @@
+default: all
+
+all:
+	go generate
+	go build
+
+.PHONY: install
+install:
+	go install
+
+.PHONY: clean
+clean:
+	rm qrc.go
+	go clean
```

Comment:
"Makefile" is the name recommended by GNU Make, let's stick to that. I also think we don't need .PHONY targets unless we could have name collisions. Or is this good practice?